webpack/components/SnapshotManagement/components/BulkActions/BulkCreateSnapshotMenuItem/BulkCreateSnapshotMenuItem.js

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from 'react';
+import { MenuItem } from '@patternfly/react-core';
+import { ForemanHostsIndexActionsBarContext } from 'foremanReact/components/HostsIndex';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+import useSnapshotFormModal from '../../SnapshotFormModal/useSnapshotFormModal';
+
+const BulkCreateSnapshotMenuItem = () => {
+  const { selectedCount = 0, setMenuOpen } =
+    useContext(ForemanHostsIndexActionsBarContext) || {};
+  const { setModalOpen } = useSnapshotFormModal();
+
+  const handleClick = () => {
+    setMenuOpen && setMenuOpen(false);
+    setModalOpen(true);
+  };
+
+  return (
+    <MenuItem
+      itemId="bulk-create-snapshot"
+      isDisabled={selectedCount === 0}
+      onClick={handleClick}
+    >
+      {__('Create Snapshots')}
```

Comment:
```suggestion
      {__('Create snapshots')}
```
If I look at the labels of the other actions, we should use a lower-case 's' here.